batchlib/analysis/cell_level_analysis.py

```diff
@@ -555,12 +577,14 @@ def load_infected_and_control_indicators(self, in_path):
     # - number of cells
     # - cell size distribution
     # - negative ratios
-    def check_for_outlier(self, image_name, values, value_names):
+    @lru_cache(maxsize=None)
+    def check_for_outlier(self, in_file):
```

Comment:
Initially I passed `values` and `value_names` here, so we could also do image level QC based on some heuristics in this function.
This is probably not really necessary and should rather be moved to a separate job if we still want to do this to keep this job more tidy.
BUT, if we do this, we can get rid of the `outlier_type` here, because it will always be set to `manual`.

Reply:
As discussed I will leave it as is for now, we can revisit this if / when we decide to add automated image outlier detection.